### DISCUSSION TITLE
Resolving issues with Google credentials

### DISCUSSION THREAD

**Discussion by G-Ambatte** [naturalcrit/homebrewery#1580] [General]
Currently a lot of people are striking issues with their Google credentials expiring, which happens one year after the last sign in via Google. What follows are the steps required to resolve the issue.

1. Head to https://www.naturalcrit.com/login and click logout if present (in small text at the bottom of the page).
Logout location:
![image](https://user-images.githubusercontent.com/19826659/129470020-8a43b435-5189-46bc-8f66-a27b226c3b62.png)

2. Click "Sign In with Google", which will refresh the Google credentials.

3. After completing the sign in process, return to the Homebrewery page, and refresh/reload, so that it can pick up the updated credentials.

4. If this was the source of the issue, it should now be resolved and it should not return for at least twelve months.
If following these steps does not resolve the issue, please let us know!

**Comment by ghost** [comment]
These instructions resolved the issue! Thank you very much for the timely and helpful advice.

**Comment by Natsu2886** [comment]
Hi, I'm currently having this issue, and even though I did exactly as you said, it just didn't work. This saving issue only happens when trying to create a new homebrew, the biggest problem is that not only I can't save on google but I can't save it at all, even though since the bug started the old ones could be saved in the homebrewry.

**Reply by G-Ambatte** [reply]
Did you refresh the Homebrewery page, so that it could pick up the new cookie? I've added it as step 3, because without it, it will keep attempting to use the original credentials.

**Reply by Natsu2886** [reply]
I'm really sorry, I just realized that my google drive was with it's storage full, after I dealt with it the problem was solved, thanks for the help anyhow.
